agents/agents-core/src/commonMain/kotlin/ai/koog/agents/core/feature/model/FeatureStringMessage.kt

```diff
@@ -0,0 +1,45 @@
+package ai.koog.agents.core.feature.model
+
+import ai.koog.agents.core.feature.message.FeatureMessage
+import kotlinx.datetime.Clock
+import kotlinx.serialization.Serializable
+
+/**
+ * Represents a detailed implementation of [ai.koog.agents.core.feature.message.FeatureMessage] that encapsulates a string message.
+ *
+ * This class associates a string content with a specific feature-related message type, along with
+ * a timestamp indicating when the message was created.
+ *
+ * It is primarily used for text-based feature messages and integrates with the [ai.koog.agents.core.feature.message.FeatureMessage]
+ * interface to define its structure.
+ *
+ * Instances of this type are timestamped at the moment of their creation, ensuring consistent
+ * temporal tracking for feature messages.
+ *
+ * @property message The textual message content encapsulated by this feature message.
+ */
+@Serializable
+public data class FeatureStringMessage(val message: String) : FeatureMessage {
+
+    /**
+     * The timestamp, represented in milliseconds since the epoch, indicating when the
+     * feature message was created.
+     *
+     * This property provides a temporal reference that can be used to track or order
+     * the occurrence of feature messages or events. The timestamp is generated using
+     * the system clock at the time of initialization.
+     */
+    override val timestamp: Long = Clock.System.now().toEpochMilliseconds()
```

Comment:
Here and below: I'd prefer not to have `val` which is always initializes with `now()`, not only from the standpoint of side effects purity, but also testability.